main.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+    <title>Communet</title>
+    <link rel="stylesheet" href="style.css"> <!--to connect to the style.css file-->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"/>
```

Comment:
this isn't a frontend course, but IMO bootstrap is unneccessary here. You could do everything you need with some [css grid](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids)